samples/orleans/OrleansAppHost/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "Azure": {
```

Comment:
Remove these values from the samples or at least don't check-in the harded coded values.